structure/extension/gx-name-part.yaml

```
@@ -0,0 +1,95 @@
+%YAML 1.2
+---
+lang: en-US
+
+type: structure
+
+uri: https://tychonievich.github.io/gx-g7-names#name-part
+
+extension tags:
+  - _N_PART
+
+specification:
+  - A part of a name.
+  - |
+    This structure is intended to be the GEDCOM 7 parallel of the GEDOCM-X
+    element http://gedcomx.org/v1/NamePart
+
+    The preferred order of name-part structures is the preferred display order
+    of the name parts.
+  - |
+    To be fully interoperable with other parts of GEDCOM 7, applications should
+    impose the following limitations on users. For the purpose of these
+    limitations, a set of name parts are said to be contiguous if, after
+    removing any hidden parts, they are adjacent in the list of name parts.
+
+    *   Some applications (and some governments; see, e.g.
+        https://en.wikipedia.org/wiki/Rufname) require at most one primary
+        given name. This means that if there is more than one part with
+        qualifier Primary and type Given, all such parts must be contiguous.
+
+        In most cases, a single Primary Given part is adequate, but there may
+        be cases when additional details (such as particles, root names, etc)
+        will be added about parts of the primary given name, necessitating
+        breaking it up into several parts.
+
+    *   The g7:type-Name datatype requires at most one primary surname, as
+        identified by slashes in the payload. This means that
+
+        *   If there is more than one part with qualifier Primary and type
+            Surname, all such parts must be contiguous.
+
+        *   If there is no part with Primary and type Surname, then all
+            parts with type Surname must be contiguous.
+
+        In some cases a single Primary Surname part is adequate, but there
+        may be cases when additional details (such as particles, root names,
+        etc) will be added about parts of the primary surname name,
+        necessitating breaking it up into several parts.
+  - |
+    It is intended that name-part be used in such a way that all other elements
+    of a g7:INDI-NAME or g7:NAME-TRAN can be reconstructed unambiguously from
+    the name-parts alone. To achieve this, every character in the name's
+    displayed form should be in a name-part. This may mean there are name-parts
+    with no type or qualifier that store spacing, punctuation, etc.
+
+    To populate the other name data from name parts,
+
+    * Create the g7:type-NAME payload by
+
+        1.  Concatenating all name-part payloads that do not have the hidden
+            qualifier.
+        2.  Removing any U+002F characters (slash or solidus, /), which
+            g7:type-NAME cannot represent.
+        3.  Possibly delimit a part of the string with slashes:
```

Grammar doesn't match previous bullets ("concatenating", "removing"...)
```suggestion
        3.  Possibly delimiting a part of the string with slashes:
```